packages/sources/src/styles/cardSelect.scss

```diff
@@ -50,3 +50,7 @@ $card-height: 90px;
       text-overflow: ellipsis;
     }
 }
+
+.ins-c-sources__wizard--step-text {
+  max-width: 536px;
```

Comment:
Is there any specific reason for static value? I would prefer some relative value based on parent element.

Reply:
^ @beccaglass

Reply:
This adheres to PF guidelines from my understanding. It should be built into the empty state component.

Reply:
Alright. I would expect there would be some sort of variable? (--pf-c-empty-state__content--MaxWidth would be my guess)

Reply:
This is what I'm seeing in PF docs:
https://www.patternfly.org/v4/documentation/core/components/emptystate

pf-c-empty-state__body

Reply:
Nope, it was a good guess. I am just confused about the value
```css
.pf-c-empty-state__content {
    max-width: var(--pf-c-empty-state__content--MaxWidth); <-- is none
}
```
@beccaglass do you happen to know if this is the correct value?

Reply:
I don't see any width attribute on `pf-c-empty-state__body`. Anyway, I don't want to be pedantic. You probably know better than me.

Reply:
I am not sure if I believe PF variables anymore, after the changes of success color... :smiley:

Reply:
All I'm seeing in the PF template is that text wraps at 536px... no mention of a variable :/